### PR TITLE
Main 4.6.0

### PR DESCRIPTION
**Pull request by ScottBailey**
Updated version.
Updated patches.
Removed numpy pins in run sections to allow use of later versions.

### CODE REVIEW

File: recipe/conda_build_config.yaml [hunk 1]
```diff
@@ -6,8 +6,8 @@ fortran_compiler_version:  # [linux or osx]
   - 11.2.0                 # [linux or osx]
```
Comment (cbouss) [inline recipe/conda_build_config.yaml]: conda_build_config.yaml can be removed as all these versions are pinned on aggregate cbc.

**Review by cbouss — COMMENTED** [review]
That looks good to me.
I checked opencv sources and believe removing numpy pinnings won't cause issues.

**Review by cbouss — APPROVED** [review]
LGTM

**Review by varlackc — APPROVED** [review]
LGTM

**Review by skupr-anaconda — APPROVED** [review]

**Pull request merged**